core/Http.php

```diff
@@ -775,9 +773,7 @@ public static function sendHttpRequestBy(
             @fclose($file);
 
             $fileSize = filesize($destinationPath);
-            if ($contentLength > 0
-                && $fileSize != $contentLength
-            ) {
+            if ($contentLength > 0 && $fileSize !== (int) $contentLength) {
```

Comment:
`$contentLength` is now casted before, so this is obsolete

```suggestion
            if ($contentLength > 0 && $fileSize !== $contentLength) {
```